package.json

```diff
@@ -20,12 +20,12 @@
     "clean": "rimraf dist",
     "lint": "eslint src/js",
     "build:types": "tsc",
-    "build:esm": "esbuild src/js/*.js src/js/*/*.js src/js/*/*/*.js --target=es2019 --format=esm --outdir=dist",
+    "build:esm": "cp -R src/js dist",
```

Comment:
I think it's still worth using esbuild to compile away the newest JS syntax to be a bit more compatible.

Reply:
yes, this is done because esbuild will always strip the comments (no way around that :() 
which are needed for customelement manifest analyzer. 

I guess we can copy / build manifest and then build esm. might be an ok workaround.

Reply:
yeah, jsdoc generally ignores if it isn't `/**`. And not sure if tsc and the web component analyzer will pick it up as well.

Reply:
nice!